### PR TITLE
More What's New: DbContext APIs and behaviors

### PR DESCRIPTION
**Pull request by ajcvickers**

### CODE REVIEW

File: entity-framework/core/what-is-new/ef-core-7.0/whatsnew.md [hunk 1]
```diff
@@ -3422,15 +3422,15 @@ EF Core can use the same CLR type for multiple different entity types. These are
 -->
 [!code-csharp[BuildMetadata](../../../../samples/core/Miscellaneous/NewInEFCore7/DbContextApiSample.cs?name=BuildMetadata)]
 
-Notice that the entity type is named. These entity types are then accessed using a `DbSet` for the entity type, using the same name. For example:
+Notice that the shared-type entity type must be named--in this case, the name is "BuildMetadata". These entity types are then accessed using a `DbSet` for the entity type which is obtained using the name. For example:
```
Comment (AndriySvyryd) [inline entity-framework/core/what-is-new/ef-core-7.0/whatsnew.md:3425]:
```suggestion
Notice that the shared-type entity type must be named - in this case, the name is `BuildMetadata`. These entity types are then accessed using a `DbSet` for the entity type which is obtained using the name. For example:
```

File: entity-framework/core/what-is-new/ef-core-7.0/whatsnew.md [hunk 2]
```diff
@@ -3489,7 +3489,7 @@ The log level can be changed back to `Information` if desired:
 
 ### `IEntityEntryGraphIterator` is publicly usable
 
-In EF,  the <xref:Microsoft.EntityFrameworkCore.ChangeTracking.IEntityEntryGraphIterator> service can be used by applications. This is the service used internally when discovering a graph of entities to track, and also by <xref:Microsoft.EntityFrameworkCore.ChangeTracking.ChangeTracker.TrackGraph%2A>. Here's an example that iterates over all entities reachable from some starting entity:
+In EF7, the <xref:Microsoft.EntityFrameworkCore.ChangeTracking.IEntityEntryGraphIterator> service can be used by applications. This is the service used internally when discovering a graph of entities to track, and also by <xref:Microsoft.EntityFrameworkCore.ChangeTracking.ChangeTracker.TrackGraph%2A>. Here's an example that iterates over all entities reachable from some starting entity:
```
Comment (AndriySvyryd) [inline entity-framework/core/what-is-new/ef-core-7.0/whatsnew.md:3492]: Did we discuss "EF7" vs "EF 7"?

File: entity-framework/core/what-is-new/ef-core-7.0/whatsnew.md [hunk 3]
```diff
@@ -3313,3 +3313,217 @@ LEFT JOIN (
...
+
+> [!TIP]
+> This pattern originated in the days when C# auto-properties were very limited. With modern C#, consider making the auto-properties read-only, and then either initialize them explicitly in the `DbContext` constructor, or obtain the cached `DbSet` instance from the context when needed. For example, `public DbSet<Blog> Blogs => Set<Blog>()`.
+
+### Distinguish cancellation from failure in logs
+
+Sometimes an application will explicitly cancel a query or other database operation. This is usually done using a <xref:System.Threading.CancellationToken> passed to the method performing the operation.
+
+In EF Core 6.0, the events logged when an operation is canceled are the same as those logged when the operation fails for some other reason. EF7 introduces new log events specifically for canceled database operations. These new events are, by default, logged at the <xref:Microsoft.Extensions.Logging.LogLevel.Debug> level. The following table shows the relevant events and their default log levels:
```
Comment (AndriySvyryd) [inline entity-framework/core/what-is-new/ef-core-7.0/whatsnew.md:3352]:
```suggestion
In EF Core 6, the events logged when an operation is canceled are the same as those logged when the operation fails for some other reason. EF7 introduces new log events specifically for canceled database operations. These new events are, by default, logged at the <xref:Microsoft.Extensions.Logging.LogLevel.Debug> level. The following table shows the relevant events and their default log levels:
```

**Review by AndriySvyryd — APPROVED** [review]

File: entity-framework/core/what-is-new/ef-core-7.0/whatsnew.md (hunk 2 above)
Comment (ajcvickers) [reply entity-framework/core/what-is-new/ef-core-7.0/whatsnew.md:3492]: I don't think so. EF7 comes after EF6. 🦆

**Pull request merged**